datafusion/optimizer/src/lib.rs

```diff
@@ -40,6 +40,7 @@
 pub mod analyzer;
 pub mod common_subexpr_eliminate;
 pub mod decorrelate;
+pub mod decorrelate_lateral_join;
 pub mod decorrelate_predicate_subquery;
```

Comment:
It would be really nice to figure out how to combine these passes into one unified set of decorrelation code -- now we have three passes

FYI @irenjj